src/OrchardCore/OrchardCore.Localization.Abstractions/CultureAspect.cs

```diff
@@ -7,11 +7,9 @@ namespace OrchardCore.Localization
     /// </summary>
     public class CultureAspect
     {
-        private static readonly CultureInfo _defaultCulture = CultureInfo.InstalledUICulture ?? CultureInfo.InvariantCulture;
-
         /// <summary>
         /// Gets or sets the culture.
         /// </summary>
-        public CultureInfo Culture { get; set; } = _defaultCulture;
+        public CultureInfo Culture { get; set; } = CultureInfo.CurrentUICulture;
```

Comment:
@Skrypt look who did this here 🤦‍♂ but even the default behavior will return a default culture, seems we did something wrong here ;)

This mean it will assign the default culture even if the content item doesn't have a `LocalizationPart` which is wrong, it should be `null`, @sebastienros do you remember why we doing thing? Coz I'm facing an issue in Lucene module right now, but I did a workaround, sooner or later we should fix this without impact other places